packages/@react-spectrum/s2/src/Picker.tsx

```diff
@@ -480,11 +519,10 @@ export function PickerSection<T extends object>(props: PickerSectionProps<T>): R
   return (
     <>
       <AriaListBoxSection
-        {...props}
-        className={section({size})}>
```

Comment:
We don't need the section styles anymore?

Reply:
no we don't need those styles anymore. it took advantage of grid layouts which we couldn't take advantage of with the way virtualizer works